encryption/Readme.md

```diff
@@ -318,23 +302,23 @@ echo number of lines: `wc -l customers.json | awk '{print $1}'`
 echo file size: `du -sh customers.json | awk '{print $1}'`
 ```
 
-Compute the duration of sending 'customers.json' in gateway with encryption with `kafka-console-producer`
+Compute the duration of sending 'customers.json' within Gateway with encryption by using the `kafka-console-producer`;
 
 ```sh
 time docker compose exec -T kafka-client \
```

Comment:
@framiere this seemed to hang for me, can you check and modify it. Or if another way to communicate this performance hit effectively that would work.